addons/l10n_vn/data/template/account.tax-vn.csv

```diff
@@ -1,33 +1,49 @@
+"","","","","","","","","tax","invoice","+tax_purchase_import_10","chart33312",,
+"","","","","","","","","base","refund","-tax_purchase_import_10_base",,,
+"","","","","","","","","tax","refund","-tax_purchase_import_10","chart33312",,
+"tax_purchase_import_5","5% Import","Deductible VAT 5% For Imported Goods","Deductible VAT 5% For Imported Goods","5.0","percent","purchase","tax_group_5","base","invoice","+tax_purchase_import_5_base",,"False","Thuế GTGT được khấu trừ cho hàng nhập khẩu 5%"
+"","","","","","","","","tax","invoice","+tax_purchase_import_5","chart33312",,
+"","","","","","","","","base","refund","-tax_purchase_import_5_base",,,
+"","","","","","","","","tax","refund","-tax_purchase_import_5","chart33312",,
+"tax_purchase_import_0","0% Import","Deductible VAT 0% For Imported Goods","Deductible VAT 0% For Imported Goods","0.0","percent","purchase","tax_group_0","base","invoice","+tax_purchase_import_0_base",,"False","Thuế GTGT được khấu trừ cho hàng nhập khẩu 0%"
+"","","","","","","","","tax","invoice",,,,
+"","","","","","","","","base","refund","-tax_purchase_import_0_base",,,
+"","","","","","","","","tax","refund",,,,
+"tax_import","Import Tax","Import Tax","Import Tax","5.0","percent","purchase","tax_group_import","base","invoice","+tax_import_base",,"True","Thuế nhập khẩu"
+"","","","","","","","","tax","invoice","+tax_import","chart3333",,
+"","","","","","","","","base","refund","-tax_import_base",,,
+"","","","","","","","","tax","refund","+tax_import","chart3333",,
+"tax_sale_vat10","10%","Value Added Tax (VAT) 10%","Value Added Tax (VAT) 10%","10.0","percent","sale","tax_group_10","base","invoice","+Untaxed sales of goods and services taxed 10%",,"False","Thuế GTGT phải nộp 10%"
+"","","","","","","","","tax","invoice","+VAT on sales of goods and services 10%","chart33311",,
+"","","","","","","","","base","refund","-Untaxed sales of goods and services taxed 10%",,,
+"","","","","","","","","tax","refund","-VAT on sales of goods and services 10%","chart33311",,
+"tax_sale_vat5","5%","Value Added Tax (VAT) 5%","Value Added Tax (VAT) 5%","5.0","percent","sale","tax_group_5","base","invoice","+Untaxed sales of goods and services taxed 5%",,"False","Thuế GTGT phải nộp 5%"
+"","","","","","","","","tax","invoice","+VAT on sales of goods and services 5%","chart33311",,
+"","","","","","","","","base","refund","-Untaxed sales of goods and services taxed 5%",,,
+"","","","","","","","","tax","refund","-VAT on sales of goods and services 5%","chart33311",,
+"tax_sale_vat0","0%","Value Added Tax (VAT) 0%","Value Added Tax (VAT) 0%","0.0","percent","sale","tax_group_0","base","invoice","+Untaxed sales of goods and services taxed 0%",,"False","Thuế GTGT phải nộp 0%"
+"","","","","","","","","tax","invoice",,,,

```

Comment:
You lost the "" for the last few columns, would be best to stay consistent and keep them even on empty cells.
Thanks!